packages/ui-voip/src/components/VoipDialPad/VoipDialPad.spec.tsx

```diff
@@ -1,75 +1,79 @@
+/* eslint-disable no-await-in-loop */
+import { installPointerEvent, triggerLongPress } from '@react-spectrum/test-utils';
 import { mockAppRoot } from '@rocket.chat/mock-providers';
-import { fireEvent, render, screen } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import DialPad from './VoipDialPad';
 
-describe('VoipDialPad', () => {
-	beforeEach(() => {
-		jest.useFakeTimers();
-	});
+const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
 
-	afterEach(() => {
-		jest.clearAllTimers();
-	});
+installPointerEvent();
 
-	it('should not be editable by default', () => {
-		render(<DialPad value='' onChange={jest.fn()} />, { wrapper: mockAppRoot().build(), legacyRoot: true });
+beforeEach(() => {
+	jest.useFakeTimers();
+});
 
-		expect(screen.getByLabelText('Phone_number')).toHaveAttribute('readOnly');
-	});
+afterEach(() => {
+	jest.clearAllTimers();
+});
 
-	it('should enable input when editable', () => {
-		render(<DialPad editable value='' onChange={jest.fn()} />, { wrapper: mockAppRoot().build(), legacyRoot: true });
+it('should not be editable by default', async () => {
```

Comment:
Does adding `async` affect the test run in any way here _(considering no `await` is being used)_ or are you just adding it preemptively?